zeq/eventDescriptor.h

```diff
@@ -32,7 +32,10 @@ enum EventDirection
  * It contains the REST api name, the 128bit zeq event id and the schema
  * needed for automatic JSON serialization/deserialization.
  */
-struct EventDescriptor : public boost::noncopyable
+struct EventDescriptor
+#ifndef BOOST_NO_CXX11_RVALUE_REFERENCES
+    : public boost::noncopyable
+#endif
```

Comment:
Should it not stay noncopyable? To my understanding, it is moveable but not copyable.